src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -360,39 +360,48 @@ def get_top_5_issues_by_count_for_file(
 
     # filter on the subquery to squash group_ids with the same title and culprit
     # return the group_id with the greatest count of events
+    query = (
+        Query(subquery)
+        .set_select(
+            [
+                Column("function_name"),
+                Function(
+                    "arrayElement",
+                    (Function("groupArray", [Column("group_id")]), 1),
+                    "group_id",
+                ),
+                Function(
+                    "arrayElement",
+                    (Function("groupArray", [Column("event_count")]), 1),
+                    "event_count",
+                ),
+            ]
+        )
+        .set_groupby(
+            [
+                Column("title"),
+                Column("culprit"),
+                Column("function_name"),
+            ]
+        )
+        .set_orderby([OrderBy(Column("event_count"), Direction.DESC)])
+        .set_limit(5)
+    )
+
     request = SnubaRequest(
         dataset=Dataset.Events.value,
         app_id="default",
         tenant_ids={"organization_id": projects[0].organization_id},
-        query=(
-            Query(subquery)
-            .set_select(
-                [
-                    Column("function_name"),
-                    Function(
-                        "arrayElement",
-                        (Function("groupArray", [Column("group_id")]), 1),
-                        "group_id",
-                    ),
-                    Function(
-                        "arrayElement",
-                        (Function("groupArray", [Column("event_count")]), 1),
-                        "event_count",
-                    ),
-                ]
-            )
-            .set_groupby(
-                [
-                    Column("title"),
-                    Column("culprit"),
-                    Column("function_name"),
-                ]
-            )
-            .set_orderby([OrderBy(Column("event_count"), Direction.DESC)])
-            .set_limit(5)
-        ),
+        query=query,
     )
-    return raw_snql_query(request, referrer=Referrer.GITHUB_PR_COMMENT_BOT.value)["data"]
+
+    try:
+        return raw_snql_query(request, referrer=Referrer.GITHUB_PR_COMMENT_BOT.value)["data"]
+    except Exception:
+        logger.exception(
+            "github.open_pr_comment.snuba_query_error", extra={"query": request.to_dict()["query"]}
```

Comment:
```suggestion
            "github.open_pr_comment.snuba_query_error", extra={"query": request.query}
```
i think this is equivalent